third_party/terraform/tests/resource_compute_instance_migrate_test.go

```diff
@@ -13,7 +13,9 @@ import (
 	"github.com/hashicorp/terraform/terraform"
 )
 
-func TestComputeInstanceMigrateState(t *testing.T) {
+func TestAccComputeInstanceMigrateState(t *testing.T) {
```

Comment:
Are these tests actually working now? Last I ran these I went through ~5 rounds of fixes before I gave up and filed: https://github.com/terraform-providers/terraform-provider-google/issues/2957

Reply:
They are!